uefi.adoc

```diff
@@ -1,37 +1,52 @@
 [[uefi]]
 == UEFI Requirements
-=== UEFI Version
-=== UEFI Compliance
-=== UEFI Platform Initialization
-The RISC-V Boot and Runtime Services Specification does not require following
-the UEFI Platform Initialization Specification cite:[UEFI-PI].
-
-Implementing the following elements is recommended:
-
-* Security Architectural Protocol (EFI_SECURITY_ARCH_PROTOCOL)
-* Security2 Architectural Protocol (EFI_SECURITY2_ARCH_PROTOCOL)footnote:[The
-Security and Security2 Architectural Protocols are overridden by some
-bootloaders (e.g. systemd-boot) to validate EFI binaries that cannot be
-validated against the UEFI security database.]
-
-=== UEFI System Environment and Configuration
-==== Privilege Levels
-===== UEFI Boot in HS-Mode
-===== UEFI Boot in (V)S-Mode
-==== System Volume Format
-==== UEFI Image Format
-===== UEFI Drivers
-===== UEFI Applications
-===== PE/COFF Images
-==== Graphics Output (GOP) Protocol
-==== Address Translation Support
-=== UEFI Boot Services
-==== Memory Map
-==== UEFI Loaded Images
-==== Configuration Tables
+
+The Unified Extensible Firmware Interface (UEFI) specification describes the interface between the OS and the supervisor-mode firmware.
+
+This section defines the BRS-I mandatory and optional UEFI requirements on top of cite:[UEFI], and is optional and recommended for BRS-B. Additional non-normative guidance may be found in the <<uefi-guidance, appendix>>.
+
+A compliant system must:
+
+* Implement a 64-bit UEFI firmware.
+* Meet the 3rd Party UEFI Certificate Authority (CA) requirements on UEFI memory mitigations cite:[MSUefiCaRequirements].
+* Meet the following UEFI memory map requirements:
+** The default memory space attribute must be EFI_MEMORY_WB.
+** Enable address translation.
+*** Any memory space defined by the UEFI memory map must be identity mapped (virtual address equals physical address), although the attributes of certain regions may not have all read, write and execute attributes or be unmarked for purposes of platform protection.
+*** The mappings to other regions are undefined and may vary from implementation to implementation.
+*** The highest hart-supported MMU configuration must be used. E.g., Sv39 should not be used on an Sv48-capable chip.
```

Comment:
What is the motivation for this? I don't quite understand the motivation for this.

Reply:
Applies to DXE Boot Services only. Implied motivation/assumption is using for probing purposes.  Claim is that MMU capability is hard to discover.  https://docs.google.com/document/d/1zSw9eaW3tlrdGxkclo9FsLvYxUvhAjRCR8_71EfRJR4/edit

Reply:
Exists in RHCT.